RemoveDup/function.js

```diff
@@ -0,0 +1,22 @@
+function removeDuplicates(arr) {
```

Comment:
Please add description for function with params 
[Ref for doc](https://dev.to/paulasantamaria/document-your-javascript-code-with-jsdoc-2fbf)